base/src/main/proto/spine/options.proto

```diff
@@ -363,15 +363,28 @@ extend google.protobuf.MessageOptions {
     // release.
     bool beta_type = 73914;
 
-    // Specifies the name of a Java interface to be implemented by this message.
+    // Specifies a characteristic inherent in the the given message type.
     //
-    // The interface is generated into the Protobuf generation destination directory. The interface
-    // itself is a marker interface derived from `com.google.protobuf.Message`. Unless the interface
-    // is specified by its FQN, it will have the same Java package as the message.
+    // Example:
+    // ```
+    // message CreateProject {
+    //     option (is) = "CommandMessage";
```

Comment:
`CommandMessage` is going to be a built-in interface. How about having `ProjectCommand` instead?